Source/LibationFileManager/TemplateTags.cs

```diff
@@ -47,6 +47,7 @@ private TemplateTags(string tagName, string description, string defaultValue = n
         public static TemplateTags DateAdded { get; } = new TemplateTags("date added", "Date added to your Audible account. e.g. yyyy-MM-dd", $"<date added [{DEFAULT_DATE_FORMAT}]>", "<date added [...]>");
         public static TemplateTags IfSeries { get; } = new TemplateTags("if series", "Only include if part of a book series or podcast", "<if series-><-if series>", "<if series->...<-if series>");
         public static TemplateTags IfPodcast { get; } = new TemplateTags("if podcast", "Only include if part of a podcast", "<if podcast-><-if podcast>", "<if podcast->...<-if podcast>");
+        public static TemplateTags IfPodcastParent { get; } = new TemplateTags("if podcastparent", "Only include if item is a podcast series parent", "<if podcastparent-><-if podcastparent>", "<if podcastparent->...<-if podcastparent>");
```

Comment:
I love seeing good architectural decisions pay dividends like this. It's so clean! My only suggestion is to also update the tagging documentation.

Reply:
Never mind. I see it now. Not sure how I missed that before. Whoops